src/app/lib/utilities/ichefURL/index.js

```diff
@@ -2,28 +2,27 @@
 const WEBP_ORIGIN_CODES = ['cpsdevpb', 'cpsprodpb'];
 
 const buildPlaceholderSrc = (src, resolution) => {
-  const parts = src.split('/');
-  const [domain, media, imgService, width, ...extraParts] = parts;
-  const definedWidth = width.replace('$width', resolution);
-  const domainWithProtocol = `https://${domain}`;
-
+  // return urn schemes unmodified
+  if (src.includes('urn:')) return src;
```

Comment:
Can we make this comment unnecessary?

```suggestion
  const isUrnScheme = src.includes('urn:')

  if (isUrnScheme) {
    return src;
  }
```

Reply:
Sure we can.